frontend/src/app/task-area/task-area.component.html

```diff
@@ -1,14 +1,22 @@
 <div class="task-area">
-  <h1>All Tasks</h1>
-  <input class="new-input" type="text" placeholder="Add a new task insdie ‘All’ category" />
+  <h1>{{ currentTab }} Tasks</h1>
+
+  <input (keyup.enter)="createTask()" [formControl]="taskText" class="new-input" type="text"
```

Comment:
```suggestion
  <input
      (keyup.enter)="createTask()"
      [formControl]="taskText"
      class="new-input"
      type="text"
```

Reply:
done

Reply:
I still see previous formatting. That's strange (